src/encord_active/cli/main.py

```diff
@@ -73,23 +75,105 @@ def download(
     from encord_active.lib.project.sandbox_projects import fetch_prebuilt_project
 
     project_path = fetch_prebuilt_project(project_name, project_dir)
+    success_with_visualise_command(project_path, "Successfully downloaded sandbox dataset. ")
 
-    cwd = Path.cwd()
-    cd_dir = project_path.relative_to(cwd) if project_path.is_relative_to(cwd) else project_path
 
-    rich.print(
-        Panel(
-            f"""
-Successfully downloaded sandbox dataset. To view the data, run:
+@cli.command(
+    name="init",
+)
+def import_local_project(
+    root: Path = typer.Argument(
+        ...,
+        help="The root directory of the dataset you are trying to import",
+        file_okay=False,
+    ),
+    glob: List[str] = typer.Option(
+        ["**/*.jpg", "**/*.png", "**/*.tiff"],
+        "--glob",
+        "-g",
+        help='Glob pattern to choose files in the "leaf directories". Note that you can repeat the `--glob` argument if you want to match multiple things.',
+    ),
+    target: Path = typer.Option(
+        Path.cwd(),
+        "--target",
+        "-t",
+        help="Directory where the project would be saved.",
+        file_okay=False,
+    ),
+    project_name: str = typer.Option(
+        "",
+        "--name",
+        "-n",
+        help="Name to give the new project. If no name is provided, the root directory will be used with '[EA] ' prepended",
+    ),
+    symlinks: bool = typer.Option(
+        False,
+        help="Use symlinks instead of copying images to the target directory.",
+    ),
+    dryrun: bool = typer.Option(
+        False,
+        help="Print the files that will be imported WITHOUT importing them.",
+    ),
+):
+    """
+    [bold]Initialise[/bold] a project from your local file system by searching for images based on the `glob` arguments.
+    By default, all jpeg, jpg, png, and tiff files will be matched.
+
+
+    """
+    from encord_active.lib.project.local import init_local_project
 
-[cyan]cd "{escape(cd_dir.as_posix())}"
-encord-active visualise
-    """,
-            title="🌟 Success 🌟",
-            style="green",
-            expand=False,
+    try:
+        project_path = init_local_project(
+            root=root, target=target, glob=glob, project_name=project_name, symlinks=symlinks, dryrun=dryrun
         )
-    )
+        if dryrun:
+            if not isinstance(project_path, list):
+                raise ValueError("Expected a list of paths for the dryrun execution")
```

Comment:
[curious]🤔

Why are we expecting a list if its a dry run but a path if not?

Reply:
This is because if we do a dryrun, we want to print all the stuff, but I didn't want the printing to be in the lib. I wanted that to be in the CLI. So in order to be able to print int, I had to return the list of files.

Perhaps there is a better way to do this? 🤷🏻

Reply:
Ok, I understand the reasoning now. Printing in the lib isn't what we want but we do want consistency between dry runs and regular runs.

There might be a better way to decouple this. If we move the parts that are writing to the disk into a separate `save` operation (another method/return a callback/create a class that wraps the behaviour) we can avoid some of the complexity and we won't need to pass the `dryrun` outside of the cli.

Let me know what you think, I realise this might not be the smallest change, and probably not critical for now.